.env

```diff
@@ -0,0 +1 @@
+FIREBASE_API_KEY = AIzaSyAQesobMMTDnLrn2U8WyFIQN5lyzTrAHrs
```

Comment:
You should never push your .env on GIT, because the main purpose of .env is to avoid pushing confidential detail.
So please remove this.